src/app/rankings/components/ranking-user-data/ranking-user-data.component.html

```diff
@@ -0,0 +1,47 @@
+<div *ngIf="itsMe" class="itsMeDiv">
```

Comment:
Não entendi porque você ta fazendo duas vezes as tres melhores posições. Você pode explorar um negócio cahamado [ngClass]
Nele você consegue passar uma classe como variável:
[ngClass] ="profileClass"

No ts você faz toda a lógica para decidir qual(is) será(ão) a(s) class(s):
itsMe? Então a classe vai ser this.profileClass="itsMe"
É ouro? Então a classe vai ser this.profileClass = this.profileClass + 'position__gold'
É prata? Então a classe vai ser this.profileClass = this.profileClass + 'position__silver'

Reply:
Com isso você evita código repetido